buildenv/Build_Instructions_V8.md

```diff
@@ -386,6 +404,48 @@ bash configure --disable-ccache \
 
 :pencil: Modify the paths for freemarker and freetype if you manually downloaded and unpacked these dependencies into different directories.
 
+2) Win 32-bit
+[1] Modify the script to support the 32-bit setting
+```
+openj9-openjdk-jdk8\jdk\make\closed\autoconf\generated-configure.sh
+  if test "x$OPENJ9_CPU" = xx86-64; then
+    if test "x$OPENJDK_BUILD_OS" = xlinux; then
+      OPENJ9_PLATFORM_CODE=xa64
+    elif test "x$OPENJDK_BUILD_OS" = xwindows; then
+      OPENJ9_PLATFORM_CODE=wa64
+      if test "x$OPENJ9_LIBS_SUBDIR" = xdefault; then
+        OPENJ9_BUILDSPEC="win_x86-64"
+      if test "x$OPENJDK_TARGET_CPU_BITS" = x32; then  <----- for 32bit setting
+        OPENJ9_PLATFORM_CODE=wi32
+        OPENJ9_BUILDSPEC="win_x86"
+      fi
+    else
+        OPENJ9_BUILDSPEC="win_x86-64_cmprssptrs"
+    fi
+```
+
+[2] Specify the path to load jvm.dll for java lanncher
+```
+./jdk/make/CopyFiles.gmk
+  $(JVMCFG):
+	$(MKDIR) -p $(@D)
+	$(RM) $(@)
+        # Now check for other permutations
+        ifeq ($(JVM_VARIANT_SERVER), true)
+	  $(PRINTF) "-j9vm KNOWN\n">>$(@)  <-------------
+		$(PRINTF) "-client IGNORE\n">>$(@)
+		$(PRINTF) "-server IGNORE\n">>$(@)
+          ifeq ($(JVM_VARIANT_MINIMAL1), true)
+	    $(PRINTF) "-minimal KNOWN\n">>$(@)
+          endif
+        else
+```
+
+[3] run configure script as follows:
+```
+bash configure --disable-ccache --with-boot-jdk=/cygdrive/c/temp/jdk7 --with-freemarker-jar=/cygdrive/c/temp/freemarker.jar --with-freetype-include=/cygdrive/c/temp/freetype/include --with-freetype-lib=/cygdrive/c/temp/freetype/lib32  --with-conf-name=windows-x86-normal-release  --target=x86 --with-target-bits=32 --with-noncompressedrefs
```

Comment:
Are all of these required?  I would have thought `-with-target-bits=32` would make `--with-noncompressedrefs` redundant

Reply:
I need to double-check whether "--with-noncompressedrefs" is mandatory but the rest of options are required for 32-bit.

Reply:
"--with-noncompressedrefs" must be specified otherwise it will up with a bunch of compilation failures such as " error C2371: 'size_t' : redefinition; different basic type", etc.

Reply:
After everything else is done, we should change the code so --with-noncompressedrefs is automatic when -with-target-bits=32 is specified. So in the end this should not be required.

Reply:
and also check if we can simplify anything else so not all of `-with-conf-name=windows-x86-normal-release  --target=x86 --with-target-bits=32` is required. Although this is the last priority.

Reply:
-with-conf-name=windows-x86-normal-release <-----  you need to specify a name for your build; otherwise, the name will end up being 64-bit name automatically picked up by build. e.g windows-x86_64-normal-server-release

Both --target=x86  both --with-target-bits=32 are the most important options to set up all 32-bit related configuration unless we modify all the scripts related to unify these option.

I used to try "--openjdk-target=i586-pc-cygwin" to cover everything but it ended up with failure to generate a certain .exe file during compilation.